<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileComparatorWindow</class>
 <widget class="QDialog" name="FileComparatorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File Comparator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>First File:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblFirstFile">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnFirstFile">
         <property name="text">
          <string>Load New File</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Second File:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblSecondFile">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSecondFile">
         <property name="text">
          <string>Load New File</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="treeDetails">
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnSaveDetails">
     <property name="text">
      <string>Save Details to File</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
